app/migrations/0003_tweet.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.1 on 2016-10-30 23:14
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('app', '0002_auto_20161030_1512'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='Tweet',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('text', models.CharField(max_length=256)),
```

Comment:
I'm not exactly sure how special characters are encoded in the tweets (what we receive _might_ be more than 140 characters). That, and I forgot the specific number of characters when I was choosing column sizes :P.